runtime/android/core_internal/src/org/xwalk/core/internal/XWalkContent.java

```diff
@@ -119,12 +119,16 @@ public XWalkContent(Context context, AttributeSet attrs, XWalkViewInternal xwVie
         MediaPlayerBridge.setResourceLoadingFilter(
                 new XWalkMediaPlayerResourceLoadingFilter());
 
-        setNativeContent(nativeInit());
+        setNativeContent(nativeInit(), attrs);
 
         XWalkPreferencesInternal.load(this);
     }
 
     private void setNativeContent(long newNativeContent) {
```

Comment:
`setNativeContent` is only for internal use, so only one interface is enough. There's no need to keep this interface.